### PR TITLE
use pip instead of brew for pygame install

### PR DESCRIPTION
**Pull request by Adimote**
installing pygame with brew isn't the recommended method any more
(Not sure if it was to begin with). So lets use pip.

### CODE REVIEW

**Review by trickeydan — APPROVED** [review]
Looks fine to me, but I don't think I have write access to this repo anyway so my approval probably doesn't help!

Part of me wonders if we should be encouraging installation in a virtualenv, but that's definitely out of scope for here.

**Review by PeterJCLaw — APPROVED** [review]
This seems fine to me, though I have no way to test it.
Do we know anyone with a Mac who can check that this works as expected?

File: install-macosx.command
Comment (raccube) [inline install-macosx.command:9]: 1. This installs python 3. According to the readme, this is supposed to run under Python 2. macOS ships with 2.7.10 so this could actually be skipped.
2. If you already have the package is installed this will actually fail

**Review by raccube — COMMENTED** [review]
Does in fact seem to run although I don't have a script to run it with
image

**Comment by RealOrangeOne** [comment]
@mltnhm is that on Catalina? The team which raised this issue on the forum found they needed to make this patch to make it run on Catalina, although whether it works on Mojave is still good to know!

**Comment by raccube** [comment]
@RealOrangeOne nope, Mojave (10.14.6) I did check on my install of Catalina and it has the same version.

**Comment by raccube** [comment]
However, it does seem to hang on this until the script finishes running:
Screenshot 2019-11-10 at 10 01 53 pm

**Comment by raccube** [comment]
I dug around and found this thread: https://github.com/pygame/pygame/issues/555
So I installed miniconda and it did in fact work.

**Comment by PeterJCLaw** [comment]
Ah, sounds like we need a deeper change here. @mltnhm would you be able to put together some changes to the script such that this works on the versions of macOS that you've been testing on?

**Comment by raccube** [comment]
@PeterJCLaw Sure thing! ✨

**Comment by RealOrangeOne** [comment]
Closing in favor of #6, which is fully tested on macOS.

**Pull request closed without merging**
